tox.ini

```diff
@@ -100,36 +104,6 @@ commands =
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source nidmm -m py.test src/nidmm/system_tests --junitxml=generated/junit/junit-dmm-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: python tools/install_local_wheel.py --driver nifgen
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source nifgen -m py.test src/nifgen/examples --junitxml=generated/junit/junit-fgen-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source nifgen -m py.test src/nifgen/system_tests --junitxml=generated/junit/junit-fgen-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: python tools/install_local_wheel.py --driver niscope
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source niscope -m py.test src/niscope/examples --junitxml=generated/junit/junit-scope-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source niscope -m py.test src/niscope/system_tests --junitxml=generated/junit/junit-scope-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: python tools/install_local_wheel.py --driver niswitch
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source niswitch -m py.test src/niswitch/examples --junitxml=generated/junit/junit-switch-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source niswitch -m py.test src/niswitch/system_tests --junitxml=generated/junit/junit-switch-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: python tools/install_local_wheel.py --driver nimodinst
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source nimodinst -m py.test src/nimodinst/examples --junitxml=generated/junit/junit-modinst-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --append --source nimodinst -m py.test src/nimodinst/system_tests --junitxml=generated/junit/junit-modinst-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: python tools/install_local_wheel.py --driver nise
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --source nise -m py.test src/nise/examples --junitxml=generated/junit/junit-nise-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: coverage run --rcfile=tools/coverage_system_tests.rc --source nise -m py.test src/nise/system_tests --junitxml=generated/junit/junit-nise-{envname}-{env:BITNESS:64}.xml {posargs}
-    system_tests: coverage report --rcfile=tools/coverage_system_tests.rc
-    system_tests: coverage html --rcfile=tools/coverage_system_tests.rc --directory=generated/htmlcov/system_tests
     nidcpower_system_tests: python --version
```

Comment:
Why are these lines still required here?

Reply:
I mean `<driver>_system_tests` rules

Reply:
Because the internal test suites for the drivers assume tox.ini. I think it is beyond the scope of this PR to fix all those up, though it may be something we should do if we do go with this change.

They could be removed from `tox-travis.ini` but that would mean the two files are no longer the same other than the env list.